### ISSUE TITLE
pitest-maven 1.1.1 and above lost compatibility with JDK 1.5

### ISSUE THREAD

**Issue by Sergh888** [hcoles/pitest#155]
Hello.
pitest-maven 1.1.1 and above lost compatibility with JDK 1.5.
I support the old code. Difficult get the actual code coverage (without pitest). Please return compatibility with JDK 1.5.

**Comment by hcoles** [comment]
@Sergh888 

Can you confirm if the latest 1.1.4-SNAPSHOT works for you with Java 5?

I have updated the project to target 1.5 bytecode, but am unable to test against 1.5 as cloudbees java 5 support seems to be broken.

If you can confirm this is working I will release with 1.5 support, if not the change will be rolled back.

**Comment by Sergh888** [comment]
@hcoles 

Confirm. For me, it works well.
I executed the following command:
```
mvn org.pitest:pitest-maven:mutationCoverage
```
With the following settings in the pom.xml:
```xml
<pluginRepositories>
    <pluginRepository>
        <id>ossrh</id>
        <url>https://oss.sonatype.org/content/repositories/snapshots</url>
        <snapshots>
            <enabled>true</enabled>
        </snapshots>
    </pluginRepository>
</pluginRepositories>
<build>
    <plugins>
        ...
        <plugin>
            <groupId>org.pitest</groupId>
            <artifactId>pitest-maven</artifactId>
            <version>1.1.4-SNAPSHOT</version>
            <configuration>
                <targetClasses>
                    <param>com...*</param>
                </targetClasses>
                <targetTests>
                    <param>com...*</param>
                </targetTests>
                <coverageThreshold>85</coverageThreshold>
                <mutationThreshold>85</mutationThreshold>
            </configuration>
        </plugin>
        ...
    </plugins>
</build>
```

**Comment by hcoles** [comment]
@Sergh888 thanks.

Java 5 will be supported for a few more releases. Regression is quite likely though without a way to automatically test it.

**Comment by hcoles** [comment]
Fixed in 1.1.4

**Comment by Sergh888** [comment]
@hcoles thank you so much!
Version 1.1.4 works perfectly for me.
